serializer/serializer.go

```diff
@@ -32,37 +16,6 @@ func NewSerializer(poller poller.Fetcher) *Serializer {
 
 // GetSerializedData serializes split and segment data into strings
 func (serializer *Serializer) GetSerializedData() (string, error) {
-	latestData := serializer.poller.GetSplitData()
-	if reflect.DeepEqual(latestData, poller.SplitData{}) {
-		return emptyCacheLoggingScript, nil
-	}
-	splitsData := map[string]string{}
-
-	// Serialize values for splits
-	for _, split := range latestData.Splits {
-		marshalledSplit, _ := json.Marshal(split)
-		splitsData[split.Name] = string(marshalledSplit)
-	}
-
-	marshalledSplits, err := json.Marshal(splitsData)
-	if err != nil {
-		return "", err
-	}
-
-	segmentsData := map[string]string{}
-
-	// Serialize values for segments
-	for _, segment := range latestData.Segments {
-		marshalledSegment, _ := json.Marshal(segment)
-		segmentsData[segment.Name] = string(marshalledSegment)
-	}
-
-	marshalledSegments, err := json.Marshal(segmentsData)
-	if err != nil {
-		return "", err
-	}
-
-	splitCachePreload := &SplitCachePreload{latestData.Since, latestData.UsingSegmentsCount, string(marshalledSplits), string(marshalledSegments)}
-
-	return fmt.Sprintf(formattedLoggingScript, splitCachePreload.SplitsData, splitCachePreload.Since, splitCachePreload.SegmentsData, splitCachePreload.UsingSegmentsCount), nil
+	serializedData := serializer.poller.GetSerializedData()
```

Comment:
did we talk yesterday about getting rid of the serializer and just having consumers use the poller?

Reply:
We did. After I made the decision to do a v3 branch I thought it would be easier to do that change in a subsequent PR.